course-v2/layouts/_default/baseof.html

```diff
@@ -42,6 +42,11 @@
               <div class="d-flex">
                 <div class="col-12 p-0" id="main-course-section">
                   <div class="card-body">
+                    {{ if not $isCourseHomePage }}
+                      <div id="show-desktop-course-drawer" class="show-course-info-left-arrow large-and-above-only">
+                        <img src="/images/left_arrow.svg" alt=""/>
+                      </div>
```

Comment:
**Requested Change:** This should be a button not a div (Right not it's not keyboard focusable as far as i can tell; more generally, its behavior is a button so it should be a button), and should have a reasonable aria-label. ("Show course info", perhaps?)1

Reply:
Done. Please have another look